BillTrack.Core/Contracts/SqsMessages/CreatedInvoice.cs

```diff
@@ -1,9 +1,12 @@
 using System.Text.Json.Serialization;
+using BillTrack.Core.Interfaces.Models;
 
 namespace BillTrack.Core.Contracts.SqsMessages;
 
-public class CreatedInvoice
+public class CreatedInvoice : IMessage
 {
     [JsonPropertyName("invoiceId")]
     public Guid InvoiceId { get; set; }
+
+    public string MessageType => nameof(CreatedInvoice);
```

Comment:
please, add comments in code and reference its relation with BillTrack.Application/Services/SqsMessageDispatcher.cs - for what purpose it was implemented and what it provides?